apps/server/cypress/e2e/suite1/getMxVcData.cy.ts

```diff
@@ -0,0 +1,88 @@
+describe('Create a connection in agg mode and get vc data', () => {
```

Comment:
This is pretty messy but it is working. We pretty much need to do this same test but for each different job type, and for sophtron. The sophtron vc data structure might be slightly different from MX's, so you might need to put a debugger in at the point where it gets the data to see what the attribute is for `accountGuid`